modules/cnv-edit-cdrom-vm.adoc

```diff
@@ -24,17 +24,22 @@ Use the following procedure to configure CD-ROMs for virtual machines.
 .Procedure
 
 . From the *{object-gui}* tab, select your {object}.
+
 . Select the *Overview* tab.
-. Click the pencil icon to the right of the *CD-ROMs* label to open the *Edit CD-ROM* window.
-* If no CD-ROMs are available for editing, the CD label initially displays *blank*.
-* If there are CD-ROMs available, you can eject the CD-ROM by clicking *Eject CD-ROM* or remove it by clicking *-*.
+
+. To add or edit a CD-ROM configuration, click the pencil icon to the right of the *CD-ROMs* label to open the *Edit CD-ROM* window.
```

Comment:
This sentence is a little hard to follow. I would try splitting it into two sentences.

Reply:
Modified by creating two sentences.